packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -23,14 +23,16 @@ def get_exhibitor_admin_password():
     return password
 
 
-# make the expanded config available at import time to allow determining
-# which tests should run before the test suite kicks off
-with open('/opt/mesosphere/etc/expanded.config.json', 'r') as f:
-    expanded_config = json.load(f)
-    # expanded.config.json doesn't contain secret values, so we need to read the Exhibitor admin password from
-    # Exhibitor's config.
-    # TODO: Remove this hack. https://jira.mesosphere.com/browse/QUALITY-1611
-    expanded_config['exhibitor_admin_password'] = get_exhibitor_admin_password()
+def get_expanded_config():
```

Comment:
Why we need this change?

Reply:
Can you add a commit description for https://github.com/dcos/dcos/pull/4036/commits/9408bc2646dc04ffbfd69a15bdd3989d90eee92b

Reply:
The file being opened exists only on a cluster. This change means that the file is opened only when the function is called, not on import.